src/components/Calendar/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import DateDisplay from '../DateDisplay';
+import Method from '../Method';
+import methods from '../../data/methods';
+import { getDay, incrementDate, decrementDate } from '../../helpers/dateHelper';
+import './style.css';
+
+export default class Index extends Component {
```

Comment:
Maybe we should change the class name to `Calendar`?

Reply:
Same, didn't notice :-) good catch!